install/roles/postfix/templates/master.cf

```diff
@@ -9,31 +9,26 @@
 # service type  private unpriv  chroot  wakeup  maxproc command + args
 #               (yes)   (yes)   (no)    (never) (100)
 # ==========================================================================
-smtp      inet  n       -       y       -       -       smtpd
-#smtp      inet  n       -       y       -       1       postscreen
-#smtpd     pass  -       -       y       -       -       smtpd
-#dnsblog   unix  -       -       y       -       0       dnsblog
-#tlsproxy  unix  -       -       y       -       0       tlsproxy
-subcleanup unix  n       -       -       -       0       cleanup
-  -o header_checks=pcre:/etc/postfix/headers-check.cf
-submissions inet n       -       y       -       -       smtpd
+smtp      inet  n       -       y       -       -       smtpd {{ smtpd_flags }}
+submissions inet n       -       y       -       -      smtpd {{ smtpd_flags }}
   -o syslog_name=postfix/submissions
-  -o cleanup_service_name=subcleanup
   -o smtpd_discard_ehlo_keywords=silent-discard,etrn
   -o smtpd_tls_wrappermode=yes
   -o smtpd_relay_restrictions=permit_mynetworks,permit_sasl_authenticated,reject
-{% if mail.postfix.submission.active %}
```

Comment:
Can we keep the option on `mail.postfix.submission.active`?
That's to be able to disable the `submission` port with STARTTLS and keep only the TLS-only `submissions` port (465).

Reply:
Perhaps something has been lost in translation, I will add it again, thanks for spotting this.

Reply:
Restored ;-)